modules/dice_loss_metric_notes.ipynb

```diff
@@ -108,7 +108,7 @@
    "outputs": [],
    "source": [
     "# make one hot and add batch dimension\n",
-    "make_2_class = Compose([AsDiscrete(to_onehot=2), EnsureChannelFirst()])\n",
+    "make_2_class = Compose([AsDiscrete(to_onehot=2), EnsureChannelFirst(True, \"no_channel\")])\n",
```

Comment:
Hi @wyli, it seems `grnd` already has channel dim, it maybe need to add batch dim in line136. Specifying "no_channel" here feels a little confusing to me. Thanks!

Reply:
Yes, I just changed `AddChannel` with `EnsureChannelFirst ` to remove the deprecation warnings in #919. Maybe I should remove `AddChannel` because it already has channel dim at that time.